srcpkgs/sigil/template

```diff
@@ -19,7 +19,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/Sigil-Ebook/Sigil"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=8e4919bde7b0e530e12a6feb8956e92e662270187d924a0102e505e942acacb1
+checksum=e31b7c84507ffe9ca896d8a169867923d3274c02d957bd59e5c6b1fd12b5e4c8
 python_version=3
 
 case "$XBPS_TARGET_MACHINE" in
```

Comment:
please align with what qt5-webengine's template actually does to limit building on arm

Reply:
Updated, is the change ok?

Reply:
Shouldn't it fail to build automatically once `qt5-webengine-devel` can't be found as a `makedepends`?

Reply:
@ericonr Correct, it does without adding anything. Normal xbps behavior

Reply:
I don't see why it needs "broken", then.

Reply:
@ericonr Removed broken in this update.

Reply:
@ericonr i think some maintainers like it that way as the builders bail out early and more verbose this way.
Fine with me though